common/common.go

```diff
@@ -185,3 +185,68 @@ func (l *PrivateNetList) All() bool {
 func (l *PrivateNetList) Specific(net string) bool {
 	return l.mapping[net]
 }
+
+type PrivateNetArgs struct {
+	mapping map[string]string
```

Comment:
why not just 
```go
type PrivateNetArgs map[string]string
```
?

I see the same pattern with existing `PrivateNetList` (explicit embedding and lazy initialization on Set()) - and I value in consistency (additionally I preffer xplicit `New` constructor, we don't need `.mapping == nil` checks anymore).